### ISSUE TITLE
Adjust which chapters are displayed to students and mentors

### ISSUE THREAD

**Issue by dboyer** [Iridescent-CM/technovation-app#5138]
Based on feedback from PT, this info should display for chapter selection:

1. If there are chapters in the participant's state/region, the person should see:

- The chapters in the region
- A button that says "Show all Chapters in my country" that when clicked loads a list of all chapters and the option to select one
- An option that says:  "None of the Above” Select this option if you aren’t involved with any of the Chapters listed or don’t know which to select"
<img width="633" alt="Screen Shot 2024-11-04 at 10 29 20 AM" src="https://github.com/user-attachments/assets/a52c2c39-0858-4890-bb9a-1097d69631a4">

2. If there are no chapters in the participant's state/region, the person should see:

- Text stating: There aren’t any Chapters based in your area. Click below to show all Chapters in your country.
- A button to view Chapters in the country
- An option that says "None of the Above"

<img width="714" alt="Screen Shot 2024-11-04 at 10 30 13 AM" src="https://github.com/user-attachments/assets/9ad30a51-3de6-4fe4-bd3c-79dc8bc17118">

**Comment by dboyer** [comment]
PT confirmed they would like participants to need to click a button to see all the chapters in a country, not display them automatically.

**Comment by dboyer** [comment]
Tested on preview and confirmed:

- Registered as a student in a state where the only chapter in the country was located. Did not see the button to view additional chapters in the country.
- Registered as a student in a US state with chapters. I saw three chapters in the state and had the option to view all chapters in the country. Clicking the button loaded the other chapters in the country but did not duplicate the chapters in the state.
- Registered as a student in a US state without chapters. No chapters were displayed by default and I needed to click the button to display all the chapters in the US.
- Registered as a student with no chapters in the state or country and saw the appropriate text
